### DISCUSSION TITLE
Glossary - Change

### DISCUSSION THREAD

**Discussion by muricna** [eprocurementontology/eprocurementontology#494] [Ideas]
New definition: A new document or record that reflects alterations required in the original document or record.

This definition can be applied to the corrigendum as used today in the notices or could be used for versioning.
A record is information that has been declared to be final and cannot be altered in anyway.

WG approval 28/05/2018

**Comment by JachymHercher** [comment]
Are we being 100% clear about "records" vs. "changes"? Specifically, if we say a "record" "cannot be altered", then I'm not sure that the definition of "change" should include "reflects alterations [...] in the original (?) record." 

This could work fine if we defined the difference between record and document, as hinted in #119, only as "something that cannot be deleted".

**Comment by muricna** [comment]
The definition of record has been assigned to Enric see #119

**Comment by eprocurementontology** [comment]
WG approval 25/05/2018

Term: Change
Definition: A new document or record that reflects alterations required in the original document or record.

Additional information/WG discussion points: This definition can be applied to the corrigendum as used today in the notices or could be used for versioning.
A record is information that has been declared to be final and cannot be altered in anyway.

The glossary and the conceptual model have been updated accordingly on 29/06/2018.

**Comment by JachymHercher** [comment]
My comment above has not been addressed with the "new" definition. May I ask you to reopen the issue? 

(+ formal comments: the word "required" can be dropped; "anyway" should be split in two.)

**Comment by muricna** [comment]
I would say that a record cannot be altered however a new record can show changes that changes to information need to be made.  Therefore instead of one record referring to something there are two.  An example of this is the corrigendum.  When a notice is published it is a record.

**Comment by JachymHercher** [comment]
Perhaps it is mainly a question of prepositions: "in" vs. "to". A new definition could be:

"A new document or record that reflects alterations ~~required in the~~ __to an__ original document or record."

**Comment by GiampaoloSellitto** [comment]
proposal: a record is a piece of information that must remain unchanged. Any subsequent change to a record must be evident, giving life to a new version of the record.
